From 17a8f594fa5b76035b0a5ba797b1a9d5751fce11 Mon Sep 17 00:00:00 2001
From: cnjn <1779895255@qq.com>
Date: Sat, 22 Aug 2026 20:00:23 +0800
Subject: [PATCH 16/16] arm64: defconfig: enable ZX279133 networking

Consolidate 1 validated ZX279133 development commits into one functional stage.

Original range: 88c15ca1f28369b54feb0987e36acb429d4b226a..88c15ca1f28369b54feb0987e36acb429d4b226a

Signed-off-by: cnjn <1779895255@qq.com>
---
 arch/arm64/configs/defconfig | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm64/configs/defconfig b/arch/arm64/configs/defconfig
index 5e614b8b754677ef1b6203b09802d0f113f4456a..18c3d162c0aebb60151d49a2760ac85cfa913885 100644
--- a/arch/arm64/configs/defconfig
+++ b/arch/arm64/configs/defconfig
@@ -391,6 +391,8 @@ CONFIG_DWMAC_TEGRA=m
 CONFIG_TI_K3_AM65_CPSW_NUSS=y
 CONFIG_TI_ICSSG_PRUETH=m
 CONFIG_QCOM_IPA=m
+CONFIG_ZTE_ZX279133_ETH=y
+CONFIG_ZTE_ZX279133_RTL8372N=m
 CONFIG_MESON_GXL_PHY=m
 CONFIG_AQUANTIA_PHY=y
 CONFIG_BCM54140_PHY=m
@@ -406,6 +408,7 @@ CONFIG_DP83867_PHY=y
 CONFIG_DP83869_PHY=m
 CONFIG_DP83TD510_PHY=y
 CONFIG_VITESSE_PHY=y
+CONFIG_ZX279051_PHY=y
 CONFIG_CAN_FLEXCAN=m
 CONFIG_CAN_M_CAN=m
 CONFIG_CAN_M_CAN_PLATFORM=m
@@ -413,8 +416,10 @@ CONFIG_CAN_RCAR=m
 CONFIG_CAN_RCAR_CANFD=m
 CONFIG_CAN_MCP251XFD=m
 CONFIG_MDIO_GPIO=y
+CONFIG_MDIO_ZX279133=y
 CONFIG_MDIO_BUS_MUX_MULTIPLEXER=y
 CONFIG_MDIO_BUS_MUX_MMIOREG=y
+CONFIG_PCS_XPCS=y
 CONFIG_USB_PEGASUS=m
 CONFIG_USB_RTL8150=m
 CONFIG_USB_RTL8152=m
@@ -1668,6 +1673,8 @@ CONFIG_PHY_ROCKCHIP_USBDP=m
 CONFIG_PHY_SAMSUNG_UFS=y
 CONFIG_PHY_UNIPHIER_USB2=y
 CONFIG_PHY_UNIPHIER_USB3=y
+CONFIG_PHY_ZX279133_PON_SERDES=y
+CONFIG_PHY_ZX279133_UNI_SERDES=y
 CONFIG_PHY_TEGRA_XUSB=y
 CONFIG_PHY_AM654_SERDES=m
 CONFIG_PHY_J721E_WIZ=m
